extensions/ql-vscode/test/vscode-tests/cli.ts

```diff
@@ -6,7 +6,10 @@ import { workspace } from "vscode";
  */
 function hasCodeQL() {
   const folders = workspace.workspaceFolders;
-  return !!folders?.some((folder) => folder.uri.path.endsWith("/codeql"));
+  return !!folders?.some(
+    (folder) =>
+      folder.uri.path.endsWith("/codeql") || folder.uri.path.endsWith("/ql"),
```

Comment:
```suggestion
      folder.uri.path.endsWith(`${os.sep}codeql`) || folder.uri.path.endsWith(`${os.sep}ql`),
```
I realise the original code is also using "/codeql", but since we're looking at it, should this be using `os.sep` or is forward slash always the right thing to use?

Reply:
I think the `uri` is normalized to always contain `/`. Only `fsPath` contains the platform-specific path separators.

However, could we simply use `dirname` here since that seems to be what we're checking against? Something like this (I kept it short to avoid introducing extra variables, but it might be a good idea to do explicit `===` checks instead of `includes`):

```suggestion
      ["codeql", "ql"].includes(dirname(folder.uri.fsPath)),
```

Reply:
I've made the change. I assume we're happy to assume these are file system paths?